apps/nextjs-website/src/components/atoms/WebinarHeaderBanner/WebinarHeaderBanner.tsx

```diff
@@ -12,18 +12,24 @@ import CloseIcon from '@mui/icons-material/Close';
 import EastIcon from '@mui/icons-material/East';
 import { translations } from '@/_contents/translations';
 import Link from 'next/link';
+import { Webinar } from '@/lib/types/webinar';
+import next from 'next';
 
 export type WebinarHeaderBannerProps = {
-  readonly slug: string;
-  readonly text: string;
-  readonly endDateTime: string;
+  webinars: readonly Webinar[];
 };
 
-const WebinarHeaderBanner: FC<WebinarHeaderBannerProps> = ({
-  slug,
-  text,
-  endDateTime,
-}) => {
+const WebinarHeaderBanner: FC<WebinarHeaderBannerProps> = ({ webinars }) => {
+  const nextWebinars = webinars.filter(
+    ({ startDateTime }) =>
```

Comment:
We should use `endDateTime` instead of `startDateTime` because the banner should not disappear during the webinar